common/utils/get-todays-galleries-hours.js

```diff
@@ -0,0 +1,12 @@
+import {london} from './format-date';
+
+export function getTodaysGalleriesHours (galleryOpeningHours: any) {
```

Comment:
Having `any` makes this hard to use elsewhere, but not sure we do...

Reply:
I'll put in the correct type though, definitely wouldn't hurt